.github/workflows/ci.yml

```diff
@@ -263,6 +263,11 @@ jobs:
           _make PLATFORM=nuvoton
           _make PLATFORM=d06
 
+          export ARCH=riscv
+          export CROSS_COMPILE64="ccache riscv64-linux-gnu-"
```

Comment:
I suggest you unset `CROSS_COMPILE32` here and change the prelude to arm targets build at line 90, for consistency:
```patch
           # build task
           set -e -v	          set -e -v
           export LC_ALL=C	          export LC_ALL=C
           export PATH=/usr/local/bin:$PATH  # clang
-          export CROSS_COMPILE32="ccache arm-linux-gnueabihf-"arm-linux-gnueabihf-"
-          export CROSS_COMPILE64="ccache aarch64-linux-gnu-"
           export CFG_DEBUG_INFO=n
           (...)
 
           download_plug_and_trust
           download_scp_firmware
 
+          export CROSS_COMPILE32="ccache arm-linux-gnueabihf-"arm-linux-gnueabihf-"
+          export CROSS_COMPILE64="ccache aarch64-linux-gnu-"
+
          _make	          _make
          _make COMPILER=clang